public/js/reducers/chartsReducer.js

```diff
@@ -92,29 +53,24 @@ const chartsRedux = State({
                 isStacked: state.composerVsInCopy.isStacked
             }
         };
-    },
-
-    toggleStackChart(state, isStacked) {
+    }
+    case 'GET_COMPOSER_VS_INCOPY_FAILED':
         return {
             ...state,
```

Comment:
This isn't very clear to read. Maybe make everything a bit more explicit (`Object.assign(state, composerVsIncopy: { ...etc}, {});`